include/boost/beast/http/impl/field.ipp

```diff
@@ -11,10 +11,14 @@
 #define BOOST_BEAST_HTTP_IMPL_FIELD_IPP
 
 #include <boost/beast/http/field.hpp>
+#include <boost/assert.hpp>
 #include <algorithm>
 #include <array>
 #include <cstring>
-#include <boost/assert.hpp>
+#if BOOST_BEAST_SEPARATE_COMPILATION
+#include <iostream>
```

Comment:
`<ostream>`, not `<iostream>`.

Not sure about the ifdef; the header-only config will fail in the same way without the include.

Reply:
The intent is that the user must include `<ostream>` if they want to use `operator<<` on `http::field`. For header-only, the user is the user of Beast. For the compiled lib, the user is the TU used to build the lib.

Reply:
As I said, I don't see why this code won't fail in the same way then, when the user doesn't use op<<.

Reply:
Yes I see what you mean. `<ostream>` should be included unconditionally.